js/script.js

```diff
@@ -0,0 +1,82 @@
+const todoInput = document.querySelector('#todo-input');
+const addTodoBtn = document.querySelector('#add-todo-btn');
+const todoListTitle = document.querySelector('#todo-list-title');
+const doneListTitle = document.querySelector('#done-list-title');
+const todoList = document.querySelector('.todo-list');
+const doneList = document.querySelector('.done-list');
+
+let items = [];
+
+const addNewTodo = (event) => {
+  event.preventDefault();
+  const ret = items.findIndex((item) => item.content === todoInput.value);
+  if (ret === -1 && todoInput.value) {
+    const item = {
+      key: -1,
+      content: todoInput.value,
+      isDone: false,
+    };
+    items.push(item);
+    render();
+  }
+  todoInput.value = '';
+};
+
+addTodoBtn.addEventListener('click', addNewTodo);
+
+const delTodo = (event) => {
```

Comment:
변수/함수의 이름은 길더라도 의미가 명확한 이름이 좋다고 알고 있습니다.

```suggestion
const deleteTodo = (event) => {
```